udata/templates/reuse/mail_card.html

```diff
@@ -4,7 +4,7 @@
     <table height="94" cellpadding="0" cellspacing="0" style="margin: 0;padding: 0;width: 100%;border: 1px solid #eee;">
         <tr style="margin: 0;padding: 0;">
             <td width="98" style="margin: 0;padding: 0;">
-                <img src="{{ reuse.image(94, external=True) }}" style="margin: 0;padding: 0;width: 94px;height: 94px" width="94" height="94">
+                <img src="{{ reuse.image(94, external=True)|placeholder('reuse', True) }}" style="margin: 0;padding: 0;width: 94px;height: 94px" width="94" height="94">
```

Comment:
I wonder if the inline style is still required now that we have explicit `width` and `height` attributes (not directly related but while you're at it :))